src/flint/fmpq.jl

```diff
@@ -687,6 +687,8 @@ call(a::FlintRationalField) = fmpq(fmpz(0), fmpz(1))
 
 call(a::FlintRationalField, b::Rational{BigInt}) = fmpq(b) 
 
+Base.call(::FlintRationalField, x::Rational) = Nemo.fmpq(x.num, x.den)
```

Comment:
There's no need for the Base or Nemo qualifiers here. We import Base.call and this code is inside the Nemo module.